docs/Inputs.md

```diff
@@ -127,6 +127,19 @@ import { ArrayInput, SimpleFormIterator, DateInput, TextInput } from 'react-admi
 </ArrayInput>
 ```
 
+You can also use `addButton` and `removeButton` props to pass your custom add and remove buttons to `SimpleFormIterator`.
+
+```jsx
+import { ArrayInput, SimpleFormIterator, DateInput, TextInput } from 'react-admin';
+
+<ArrayInput source="backlinks">
+    <SimpleFormIterator addButton={<CustomAddButton />} addButton={<CustomRemoveButton />}>
```

Comment:
there is a typo here, I read `addButton` twice

Reply:
Oops, we missed it. I opened the PR #5095 to fix it.